services/core-api/app/api/verifiable_credentials/manager.py

```diff
@@ -195,3 +304,104 @@ def produce_map_01_credential_payload(cls, did: str, permit_amendment: PermitAme
+            return None
+
+        untp_party_cpo = base.Party(
+            name="Chief Permitting Officer",
+            identifiers=[
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue="did:indy:candy:A2UZSmrL9N5FDZGPu68wy",
+                    identifierURI="https://candyscan.idlab.org/tx/CANDY_PROD/domain/321",
+                    verificationEvidence=base.Evidence(
+                        format=codes.EvidenceFormat.W3C_VC,
+                        credentialReference=
+                        "did:web:untp.traceability.site:parties:regulators:CHIEF-PERMITTING-OFFICER" #this is an anoncred
+                    ))
+            ])
+        orgbook_cred_url = f"https://orgbook.gov.bc.ca/entity/{orgbook_entity.registration_id}/credential/{orgbook_entity.credential_id}"
+
+        #this should have a did:web reference ideally, but orgbook doesn't have those yet.
+        untp_party_business = base.Party(
+            name=orgbook_entity.name_text,
+            identifiers=[
+                base.Identifier(
+                    scheme=ANONCRED_SCHEME,
+                    identifierValue=str(orgbook_entity.registration_id),
+                    identifierURI=orgbook_cred_url)
+            ])
+
+        facility = cc.Facility(
+            name=permit_amendment.mine.mine_name,
+            geolocation=
+            f'https://plus.codes/{plus_code_encode(permit_amendment.mine.latitude, permit_amendment.mine.longitude)}',
+            verifiedByCAB=True)
+
+        products = [
+            cc.Product(
+                name=c,
+                classifications=cc.Classification(
+                    scheme="https://unstats.un.org/unsd/classifications/Econ/cpc",
+                    classifierName=c),
+                verifiedByCAB=False) for c in permit_amendment.mine.commodities
+        ]
+
+        untp_assessments = [
+            cc.ConformityAssessment(
+                referenceRegulation=cc.Regulation(
+                    id="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/96293_01",
+                    name="BC Mines Act",
+                    issuingBody=base.Party(name="BC Government"),
+                    effectiveDate=datetime(2024, 5, 14, tzinfo=ZoneInfo("UTC")).isoformat()),
+                                                                                                   # Is there a did:web that attests to that legistlation?
+                subjectFacilities=[facility],
+                subjectProducts=products,
+                sustainabilityTopic=codes.SustainabilityTopic.Governance_Compliance)
+        ]
+        issue_date = permit_amendment.issue_date
+        issuance_date_str = datetime(
+            issue_date.year, issue_date.month, issue_date.day, 0, 0, 0,
+            tzinfo=ZoneInfo("UTC")).isoformat()
+
+        cred = cc.ConformityAttestation(
+            id="http://example.com/attestation/123",
```

Comment:
what's with the example.com URL? Mostly asking because SonarCloud is upset that it's not https

Reply:
That will the URL where the credential will be hosted, but we have no web accessible endpoint for these credentials, so it's a placeholder value.